components/Notes/Notes.js

```diff
@@ -89,26 +89,28 @@ const NotesContainer = styled.div`
   flex-direction: column;
   align-items: center;
   border-radius: 15px;
-  width: 80%;
+  width: 80vw;
+  box-shadow: var(--card-shadow-default);
+  background-color: var(--main-color3);
 `;
 
 const ShowNotesContainer = styled.div`
-  box-shadow: var(--box-shadow-default);
-  background-color: var(--bg-color1);
+  background-color: white;
   border-radius: 15px;
   margin-bottom: 15px;
   margin-top: 15px;
```

Comment:
```suggestion
margin: 15px 0;
```
You could use the short notation for margin here to reduce codelines.